modules/rosa-deleting-cluster.adoc

```diff
@@ -22,6 +29,13 @@ If add-ons are installed, the deletion takes longer because add-ons are uninstal
 ----
 $ rosa delete cluster --cluster=<cluster_name> --watch
 ----
+ifdef::sts[]
++
+[IMPORTANT]
+====
+You must wait for the cluster deletion to complete before you remove the IAM roles, policies, and OIDC provider. The account-wide roles are required to delete the resources created by the installer. The cluster-specific Operator roles are required to clean-up the resources created by the OpenShift Operators. The Operators use the OIDC provider to authenticate.
```

Comment:
I couldn't find the SG reference for this, but it is generally a best practice to include the prepositional phrase first in a sentence so that the users know the "why" before the "what". So I'd recommend: "Before you remove the IAM roles, ..., you must wait for ....".

Reply:
@lpettyjo thank you for your review and for highlighting this. I will change the sentence order accordingly and will keep this in mind in the future.

Reply:
I am reverting back to the original order of the first sentence, on the request of @0kashi.